expression/builtin_time_vec_test.go

```diff
@@ -69,7 +69,9 @@ var vecBuiltinTimeCases = map[string][]vecExprBenchCase{
 		{retEvalType: types.ETInt, childrenTypes: []types.EvalType{types.ETDatetime}},
 	},
 	ast.MicroSecond: {},
-	ast.Now:         {},
+	ast.Now: {
+		{retEvalType: types.ETInt, childrenTypes: []types.EvalType{types.ETDuration}, geners: []dataGenerator{&rangeDurationGener{0.2}}},
```

Comment:
I think the retEvalType should be types.ETDatetime and the childrenTypes should be empty. Isn't it?

Reply:
Yes, I was wrong.thank you